eng/pipelines/bicep-warnings.yml

```diff
@@ -0,0 +1,51 @@
+trigger: none
+
+pr:
+  branches:
+    include:
+      - main
+  paths:
+    include:
+      - '**/*.bicep'
+      - eng/pipelines/bicep-warnings.yml
+
+stages:
+  variables:
```

Comment:
Move this variable section above the stages:

Reply:
Done.